hack/dockerfile/binaries-commits

```diff
@@ -11,5 +11,5 @@ VNDR_COMMIT=c56e082291115e369f77601f9c071dd0b87c7120
 BINDATA_COMMIT=a0ff2567cfb70903282db057e799fd826784d41d
 
 # CLI
-DOCKERCLI_REPO=https://github.com/docker/cli
-DOCKERCLI_COMMIT=3dfb8343b139d6342acfd9975d7f1068b5b1c3d3
+DOCKERCLI_REPO=https://github.com/tonistiigi/docker-cli
+DOCKERCLI_COMMIT=18846ac45e10e214f58dab34a493045b8a7fba32
```

Comment:
It's not needed for the tests. I left it in for now so that the reviewers can easily see the actual feature.